lib/tokenizer/tokenize.js

```diff
@@ -154,10 +154,10 @@ function intoTokens(source, externalContext, internalContext, isNested) {
       // comment start within block preceded by some content, e.g. div/*<--
       metadatas.push(metadata);
       buffer.push(character);
-      buffers.push(buffer.slice(0, buffer.length - 2));
+      buffers.push(buffer.slice(0, -2));
       isBufferEmpty = false;
 
-      buffer = buffer.slice(buffer.length - 2);
+      buffer = buffer.slice(-2);
```

Comment:
Please correct me if I'm wrong but here we only take two last characters of a `buffer` while `buffer.slice(0, -2)` would take all BUT the last two.

Reply:
Indeed, got confused there when I looked at the code at the time.

Reply:
No worries! :+1: